### PR TITLE
task for schema catalog

### PR DESCRIPTION
**Pull request by Abhishek-N**

### CODE REVIEW

File: ddpui/api/airbyte_api.py [hunk 1]
```diff
@@ -632,9 +632,11 @@ def get_connection_catalog_v1(request, connection_id):
     if orguser.org.airbyte_workspace_id is None:
         raise HttpError(400, "create an airbyte workspace first")
 
-    res, error = airbytehelpers.get_connection_catalog(orguser.org, connection_id)
-    if error:
-        raise HttpError(400, error)
+    org_dict = orguser.org.__dict__
+    org_dict.pop('_state')
+    task = get_connection_catalog_task.delay(org_dict, connection_id)
+    res = task.get()
```
Comment (fatchat) [inline ddpui/api/airbyte_api.py:638]: `task.get` will block

Comment (fatchat) [reply ddpui/api/airbyte_api.py:638]: you need to return `task.id`. then you need to be able to poll the backend to find out if the task has completed. We typically do this using `TaskProgress`

**Review by fatchat — CHANGES_REQUESTED** [review]

File: ddpui/api/airbyte_api.py (hunk 1 above)
Comment (Abhishek-N) [reply ddpui/api/airbyte_api.py:636]: so we need to return both task.id and catalog response here

Comment (fatchat) [reply ddpui/api/airbyte_api.py:636]: no, the response won't be ready when `delay` returns, so you only have a task ID to return. look at how we set up a dbt workspace, or how we add custom airbyte connectors to a new airbyte workspace

**Pull request merged**